app/models/comment.rb

```diff
@@ -1,2 +1,5 @@
 class Comment < ApplicationRecord
+  belongs_to :user
+  belongs_to :post
+  has_many :likes
```

Comment:
- [ ] following the [ERD diagram](https://github.com/microverseinc/curriculum-rails/blob/main/blog-app/images/blog_app_erd_v1_1.png) the comments belong only to the user and post ,kindly fix that :pray: 

example : 


```ruby
belongs_to :user
belongs_to :post
```